ivy/functional/frontends/numpy/statistics/averages_and_variances.py

```diff
@@ -10,6 +10,70 @@
     handle_numpy_out,
 )
 
+# global
+import numpy as np
+from typing import Optional, Sequence, Union
+import ivy
+from ivy.func_wrapper import with_unsupported_dtypes
+from ivy.functional.frontends.numpy.func_wrapper import (
+    to_ivy_arrays_and_back,
+    handle_numpy_dtype,
+    from_zero_dim_arrays_to_scalar,
+    handle_numpy_out,
+)
+
+def nanmedian(
+    a,
+    /,
+     *,
```

Comment:
this line is not alligned with others, please remove that extra space before "*"

Reply:
Done